product_docs/docs/tde/15/key_stores.mdx

```diff
@@ -95,13 +95,22 @@ PGDATAKEYUNWRAPCMD='crypt decrypt aws --in %p --region us-east-1'
 
 ### Azure Key Vault example
 
-Create a key with Azure Key Vault: 
+Create a key with Azure Key Vault:
 
 ```shell
 az keyvault key create --vault-name pg-tde --name pg-tde-master-1
 ```
 
-Use the `az keyvault` command with the `pg-tde-master-1` key to wrap and unwrap the data encryption key:
+Use the `az keyvault key` command with the `pg-tde-master-1` key to wrap and unwrap the data encryption key:
+
+```shell
+PGDATAKEYWRAPCMD='az keyvault key encrypt --name pg-tde-master-1 --vault-name pg-tde --algorithm A256GCM --value @- --data-type plaintext --only-show-errors --output json | jq -r .result > "%p"'
+PGDATAKEYUNWRAPCMD='az keyvault key decrypt --name pg-tde-master-1 --vault-name pg-tde --algorithm A256GCM --value @"%p" --data-type plaintext --only-show-errors --output json | jq -r .result'
+```
+!!! Note
+    Shell commands with pipes, as in this example, are problematic because the exit status of the pipe is that of the last command. A failure of the first, more interesting command isn't reported properly. Postgres handles this somewhat by recognizing whether the wrap or unwrap command wrote nothing. However, it's better to make this more robust. For example, use the `pipefail` option available in some shells or the `mispipe` command available on some operating systems. Put more complicated commands into an external shell script or other program instead of defining them inline.
+
+Alternatively, you can use the [crypt utility](https://github.com/VirtusLab/crypt) to wrap and unwrap the data encryption key:
```

Comment:
I strongly recommend not adding a reference to this project in our official documentation. That project is no longer maintained and uses outdated dependencies that result in unexpected behaviour.